ansible/dualtor/y_cable_simulator_client.j2

```diff
@@ -0,0 +1,155 @@
+from urllib import request, error
+import json
+from sonic_py_common import logger
+
+DUTS_MAP = {
+    "{{ dual_tor_facts['positions']['upper'] }}": 0,
+    "{{ dual_tor_facts['positions']['lower'] }}": 1
+}
+
+VM_SET = "{{ testbed_facts['group-name'] }}"
+
+DUT_NAME = "{{ dut_name }}"
+
+BASE_URL = "http://{{ mux_simulator_server }}:{{ mux_simulator_port }}/"
+
+SYSLOG_IDENTIFIER = "y_cable_sim"
+helper_logger = logger.Logger(SYSLOG_IDENTIFIER)
+
+TOR_A = "tor_a"
+TOR_B = "tor_b"
+
+def _url(port):
+    """
+    Helper function to build an url for given port and target
+
+    Args:
+        port: physical port on switch, an integer
```

Comment:
Update this `port` to `physical_port` as well? Otherwise, it could be easily confused with the HTTP port.